SimConnectMSFS/WasmModuleUpdater.cs

```diff
@@ -63,58 +64,81 @@ private String ExtractCommunityFolderFromUserCfg(String UserCfg)
             char[] charsToTrim = { '"' };
             
             InstalledPackagesPath = InstalledPackagesPath.TrimEnd(charsToTrim);
-            
-            if (Directory.Exists(InstalledPackagesPath + @"\Community"))
+
+            string targetPath = Path.Combine(Path.Combine(InstalledPackagesPath, @"Community"));
+            if (Directory.Exists(targetPath))
             {
-                CommunityFolder = InstalledPackagesPath + @"\Community";
+                CommunityFolder = targetPath;
             }
 
             return CommunityFolder;
         }
+
         public bool AutoDetectCommunityFolder()
         {
-            string searchpath = Environment.GetEnvironmentVariable("AppData") + @"\Microsoft Flight Simulator\UserCfg.opt";
+            // Find the 2020 community folder
+            CommunityFolder = ExtractCommunityFolderPath(new string[] {
+                Path.Combine(Environment.GetEnvironmentVariable("AppData"), "Microsoft Flight Simulator"),
+                Path.Combine(Environment.GetEnvironmentVariable("LocalAppData"), @"Packages\Microsoft.FlightSimulator_8wekyb3d8bbwe\LocalCache\") }
+                );
+
+            // Find the 2024 community folder
+            CommunityFolder2024 = ExtractCommunityFolderPath(new string[] {
+                Path.Combine(Environment.GetEnvironmentVariable("AppData"), "Microsoft Flight Simulator 2024"),
+                Path.Combine(Environment.GetEnvironmentVariable("LocalAppData"), @"Packages\Microsoft.FlightSimulator_UNKNOWN\LocalCache\") }
```

Comment:
Packages\Microsoft.Limitless_8wekyb3d8bbwe\LocalCache

Reply:
You commented on an outdated version of this PR. It has already been merged and works fine.